index.js

```diff
@@ -6,31 +6,37 @@ const resolveAlfredPrefs = require('resolve-alfred-prefs');
 const sudoBlock = require('sudo-block');
 const plistTransform = require('./lib/plist-transform');
 const link = require('./lib/link');
+const unlink = require('./lib/unlink');
 
 // Prevent running as `sudo`
 sudoBlock();
 
-module.exports = opts => {
+const getWorkflowDir = resolveAlfredPrefs().then(prefs => path.join(prefs, 'workflows'));
+
+const readPkg = workflowDir => {
+	const cwd = process.cwd();
```

Comment:
Why are you getting the `cwd` here? Just let `readPkgUp` use its default, which is `process.cwd()`.